elk/training/train.py

```diff
@@ -48,6 +49,26 @@ def create_models_dir(self, out_dir: Path):
 
         return reporter_dir, lr_dir
 
+    def make_eval(self, model, eval_dataset):
```

Comment:
not sure if that's the best place to move the code to, everything else looks good to me.